csvcubed/shacl/v1/0/deps/dcat.json

```diff
@@ -0,0 +1,133 @@
+{
+    "@context": [
+        "shacl-context.json",
+        {
+            "this": {
+                "@id": "https://purl.org/csv-cubed/shacl/v1.0/deps/dcat.json#",
+                "prefix": true
+            }
+        }
+    ],
+    "@id": "https://purl.org/csv-cubed/shacl/v1.0/deps/dcat.json#ontology",
+    "@type": "owl:Ontology",
+    "shape": [
+        {
+            "@id": "this:CatalogDatasetShape",
+            "@type": "sh:NodeShape",
+            "targetClass": "dcat:Dataset",
+            "property": [
+                {
+                    "path": "dct:title",
+                    "name": "Title",
+                    "equals": "rdfs:label",
+                    "description": "A short title describing the catalogued item.",
+                    "datatype": "rdf:langString",
+                    "uniqueLang": true,
+                    "minCount": 1,
+                    "maxCount": 1,
+                    "minLength": 1
+                },
+                {
+                    "path": "dct:identifier",
+                    "name": "Identifier",
+                    "description": "An identifier for this catalogued item.",
+                    "datatype": "xsd:string",
+                    "minCount": 1,
+                    "maxCount": 1,
+                    "minLength": 1
+                },
+                {
+                    "path": "dct:issued",
+                    "name": "Date-time Issued",
+                    "description": "The point in time when the catalogued item was first issued.",
+                    "datatype": "xsd:dateTime",
+                    "minCount": 1,
+                    "maxCount": 1
+                },
+                {
+                    "path": "dct:modified",
+                    "name": "Date-time Modified",
+                    "description": "The point in time when the catalogued item was last updated.",
+                    "datatype": "xsd:dateTime",
+                    "minCount": 1,
+                    "maxCount": 1
+                },
+                {
+                    "path": "rdfs:comment",
+                    "name": "Comment",
+                    "description": "Short sentence describing the item being catalogued.",
+                    "datatype": "rdf:langString",
+                    "uniqueLang": true,
+                    "minCount": 0,
+                    "maxCount": 1,
+                    "minLength": 1
+                },
+                {
+                    "path": "dct:description",
+                    "name": "Description",
+                    "description": "Longer multi-paragraph description of what this catalogued item represents.",
```

Comment:
Should we mention that this is markdown?

Reply:
We haven't merged that change in yet, so I'm hesitant to bring it through - especially given we've released tooling that builds CSV-Ws of this flavour.

Once we release an update, we'll want to publish a new version of the output shapes.